content/events/agbt-2023.md

```diff
@@ -0,0 +1,30 @@
+---
+author: "AnVIL"
+conference: "AGBT"
+description: "Advances in Genome Biology and Technology (AGBT) delivers a premier experience where heads of labs, institutions, businesses, financial analysts and other high-level stakeholders come together to advance the field and drive game-changing innovation."
+eventType: "Conference"
+featured: true
+location: "Hollywood, FL, USA"
+sessions: [{sessionStart: "6 Feb 2023"}, {sessionStart: "7 Feb 2023"}, {sessionStart: "8 Feb 2023"}]
```

Comment:
Hi @jpaten would you mind adding the 9 Feb session [see Agenda](https://www.agbt.org/events/general-meeting/agenda/) here. Thank you!